tests/kernel/pipe/pipe_api/src/test_pipe_contexts.c

```diff
@@ -142,7 +142,9 @@ static void tpipe_put_small_size(struct k_pipe *ppipe, k_timeout_t timeout)
 	for (int i = 0; i < PIPE_LEN; i += wt_byte) {
 		/**TESTPOINT: pipe put*/
 		to_wt = 15;
-		k_pipe_put(ppipe, &data[i], to_wt, &wt_byte, 1, timeout);
+		zassert_false(k_pipe_put(ppipe, &data[i], to_wt, &wt_byte, 1, timeout),
+				NULL);
+
```

Comment:
zassert_false evaluates a boolean operation but k_pipe_put return an integer. Although C interprets anything != 0 as true it is not semantically correct IMHO. Would be better if you use zassert_equal here or `k_pipe_put(...) != 0`

Reply:
Thanks @ceolin , I have updated this.

Reply:
ok, just want to confirm that this is really what you want to check. Looking the test, I think you want to check that the return of `k_pipe_put` was greater than 0, since `min_xfer` is 1. Also, if the function returns a negative value in case of error you need to assert as well.

Reply:
Hi @ceolin , sorry for delay, and I want to test at least min_xfer bytes of data were read,  a negative value was returned if pipe is not working normally. So I add this assert to check. @s-kelvin , is this right?